config/schedule.rb

```diff
@@ -14,3 +16,12 @@
 every 1.day, :at => '6:00 am' do
   rake 'pqa:nightly_import'
 end
+#
+# Sanitize imported staging data
+# Follows import job staging -> production run via crontab
+# 
+every 1.day, :at => '1:00 am' do
+  rake 'db:staging:sync'
```

Comment:
The only thing that concerns me slightly is that if the rake task fails for any reason, the database is left unsanitized and we're not notified.